sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,165 @@
+from sympy import Symbol, symbols
+from sympy.physics.mechanics import (RigidBody, Particle, ReferenceFrame,
+                                     inertia)
+from sympy.physics.vector import Point, Vector
+
+__all__ = ['Body']
+
+
+class Body(RigidBody, Particle):
+    """
+    Body is a common representation of RigidBody or a Particle.
+
+    A Body represents either a rigid body or particle in classical mechanics.
+    Bodies have a body-fixed reference frame, a mass, a mass center and
+    possibly a body-fixed inertia.
+
+    Parameters
+    ----------
+    name: String
+        Defines the name of the body. It is used as the base for defining body
+        specific properties.
+    masscenter : Point, optional
+        The point which represents the center of mass of the rigid body.
+    frame : ReferenceFrame (optional)
+        The ReferenceFrame in which the rigid body is fixed.
+    mass : Sympifyable, optional
+        The body's mass.
+    body_inertia : Dyadic (instance of inertia)
+        The body's inertia about center of mass.
```

Comment:
There is a term for this: "central inertia dyadic", which means the inertia wrt to the center of mass.